apps/backend/src/routes/login/+page.svelte

```diff
@@ -0,0 +1,89 @@
+<!-- JavaScript code -->
+<script lang="ts">
+  // import for forms
+  import { createForm } from "svelte-forms-lib";
+  
+	let confirm: boolean = false;
+    
+</script>
+
+<div class="flex items-center justify-center h-screen">
+
+    <!-- Login Form - Card -->
+    <div class="w-full max-w-lg p-10 border border-primary rounded-md shadow-lg">
+
+        <!-- Login Form - Header -->
+        <p class="text-3xl font-semibold text-center mb-5">
+            Welcome Back
+        </p>
+        
+        <!-- Login Form - Form Content -->
+        <form>
+            <!-- Form Content - Email Input -->
+            <div>
+                <label class="form-control w-full max-w-md">
+                    <div class="label">
+                        <span class="label-text">
+                          Email
+                        </span>
+                    </div>
+                    <input type="text" placeholder="Name@example.com" class="input input-bordered w-full max-w-md 
+                    focus:ring-primary focus:border-primary duration-300 focus:outline-none" /> 
```

Comment:
Missing `name` attribute

Reply:
Also, space out your classes into multiple lines

Reply:
Spaced out classes. Added Name attribute from the #19 fix.